app/Post_Types/Diviner_Field/AdminModifications.php

```diff
@@ -92,37 +131,144 @@ function rc_scd_create_dashboard() {
+				</p>
+				<a href="post-new.php?post_type=diviner_field&field_type=<?php echo CPT_Field::NAME; ?>" class="button button-primary button-hero">
+					Add a New Text Field
+				</a>
+			</div>
+
+
+			<div class="field-select-wrap">
+				<h2>Date</h2>
+				<p>
+					<?php _e('Add a date field if you would like your audience to be able to filter by a date range, by year, decade, or by century. Ex: if you want to sort a collection of a thousand photos from the 20th century into decades  ' ); ?>
+				</p>
+				<a href="post-new.php?post_type=diviner_field&field_type=<?php echo Date_Field::NAME; ?>" class="button button-primary button-hero">
+					Add a New Date Field
+				</a>
+			</div>
+
+			<div class="field-select-wrap">
+				<h2>Taxonomy (Category/Tags/Keywords)</h2>
+				<p>
+					<?php _e('Add a taxonomy field for categories you want to sort your materials by (ex: by location, such as by county, by neighborhood, or by room in a museum). You will have to create the choices in this category (ex: by county; Clinton, Essex, Warren, and Jefferson). Taxonomy fields are best suited to a category with fewer than twenty choices, which do not need further explanation to a viewer.' ); ?>
+				</p>
+				<a href="post-new.php?post_type=diviner_field&field_type=<?php echo Taxonomy_Field::NAME; ?>" class="button button-primary button-hero">
+					Add a New Taxonomy Field
+				</a>
+			</div>
+
+			<div class="field-select-wrap">
+				<h2>Custom Post Type Field</h2>
+				<p>
+					<?php _e('For categories with many choices (20+) and which you would like to be able to elaborate on and attach auxiliary information, use the CPT field. A good example would be if you wished to sort your materials by their creator (photographer, author, etc.) – for each creator, this type of field allows you to create an “entry” for that creator. Other examples: donor, institution.
+
+' ); ?>
+				</p>
+				<a href="post-new.php?post_type=diviner_field&field_type=<?php echo CPT_Field::NAME; ?>" class="button button-primary button-hero">
+					Add a New Custom Post Type Field
+				</a>
 			</div>
-			<div class="wrap">
-				<?php $defaultTable ->display(); ?>
+
+			<div class="field-select-wrap">
+				<h2>Select Field</h2>
+				<p>
+					<?php _e('Add a select field to assign a piece of information that comes from a very small list of pre-set choices to each of your archive item. Examples: Art Format, with the choices being Painting, Sculpture, or Digital.  ' ); ?>
+				</p>
+				<a href="post-new.php?post_type=diviner_field&field_type=select" class="button button-primary button-hero">
+					Add a New Select Field
+				</a>
 			</div>
 
+
 		</div>
 
-		<div class="wrap wrap-diviner wrap-diviner--preset" id="wrap-diviner--preset">
+
+
+
+		<?php
+	}
+
+	function rc_scd_create_dashboard() {
```

Comment:
Is this still used? if so: why load all WordPress? My ignorance here.

Reply:
Yes, the function is used but I removed those includes.